### PR TITLE
Add cmd to get versions in GCR

### PR DESCRIPTION
**Pull request by companycy**
Provide cmd on how to get available versions. Test on ubuntu 14.04.

### CODE REVIEW

**Comment by companycy** [comment]
I signed it!

File: docs/getting-started-guides/docker.md [hunk 1]
```diff
@@ -14,7 +14,8 @@ Here's a diagram of what the final result will look like:
 
 1. You need to have docker installed on one machine.
 2. Decide what Kubernetes version to use.  Set the `${K8S_VERSION}` variable to
-   a released version of Kubernetes >= "1.2.0-alpha.7"
+   a released version of Kubernetes >= "1.2.0-alpha.7". Get all available versions by
+   ```curl -k -s -X GET https://gcr.io/v2/google_containers/hyperkube-amd64/tags/list | jq -r '.tags[]'```
```
Comment (johndmulhausen) [inline docs/getting-started-guides/docker.md]: Most systems don't have jq installed; can you do this in regex with grep -E?

Also, have a blanks space on line 18, have line 19 read "```shell", have line 20 be the command, and line 21 be "````"

Comment (companycy) [inline docs/getting-started-guides/docker.md]: Please help review new commit.

**Comment by johndmulhausen** [comment]
The curl command looks great and the docs look good too. Only question is, is 1.2.0-alpha.7 still the preferred thing we want to point users to, now that v1.2.0 is out?

**Comment by companycy** [comment]
Get your point. Please review new commit.

**Comment by johndmulhausen** [comment]
@companycy would love to merge this in but you need to resolve your merge conflict!

**Comment by johndmulhausen** [comment]
@companycy will close this PR soon if no response.

**Pull request closed without merging**